Bindings/Packages/com.virtualmaker.bindings/Runtime/Property.Extensions.cs

```diff
@@ -0,0 +1,24 @@
+#if BINDINGS_ADDRESSABLES && BINDINGS_UTIL_ASYNC && BINDINGS_UTIL_EXTENSIONS
+
+using System;
+using System.Threading.Tasks;
+using UnityEngine.ResourceManagement.AsyncOperations;
+using Utilities.Extensions;
+
+namespace VirtualMaker.Bindings
+{
+    public static class PropertyExtensions
+    {
+        public static Task<T> AwaitBindProgress<T>(this AsyncOperationHandle<T> operation, Property<float> progress, bool autoRelease = true)
```

Comment:
I don't love the coupling here between the packages. All this to save one line of code?

```
var progress = new Progress<float>(v => prop.Value = v);
await operation.AwaitWithProgress(progress);
```